src/api.py

```diff
@@ -108,38 +111,43 @@ def create_usage_report(input_text: str, for_url: str) -> UsageReport:
         operation_type=OperationType.RUN,
         operation_unit=OperationUnit.CHARACTERS,
         operation_amount=characters,
-        audit_id=for_url
+        audit_id=for_url,
     )
 
-def generate_audio(input_text: str, audit_url: str, config: ElevenlabsPluginConfig) -> (bytes, UsageReport):
+
+def generate_audio_stream(
+    input_text: str, audit_url: str, config: ElevenlabsPluginConfig
+) -> (bytes, UsageReport):
```

Comment:
Polite request to correctly type this return (it's an iterator, not bytes, right?).  On first read, I thought you were reading out all of the bytes of the response in this method, which would of course defeat the point of the streaming.